examples/neural_networks/plot_elm_diagram.py

```diff
@@ -0,0 +1,86 @@
+"""
+================
+Plot ELM diagram
+================
+
+This example plots a single-hidden layer feedforward network. It also allows
```

Comment:
I would not put a dash between single and hidden. Maybe "single-hidden-layer"  or just "single hidden layer"